src/tests/ftest/launch.py

```diff
@@ -1362,8 +1362,7 @@ def run_tests(test_files, tag_filter, args):
             # Optionally process core files
             if args.process_cores:
                 try:
-                    if not process_the_cores(
-                            avocado_logs_dir, test_file["yaml"], args):
+                    if not process_the_cores(avocado_logs_dir, test_file["yaml"], args):
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'args' in function call

Reply:
Fixed